src/celengine/catentry.cpp

```diff
@@ -80,12 +81,23 @@ bool CatEntry::isInCategory(const std::string &s) const
 
 bool CatEntry::loadCategories(Hash *hash)
 {
+    const char *d = nullptr;
+    std::string td;
+    std::string tdir;
+    hash->getString("TranslationDirectory", tdir);
```

Comment:
how can you define a directory if you don't know where your addon will be put? you should use current directory instead.

Reply:
I assumed it can be relative directory. Do you mean at every `TranslationDomain` new `bindtextdomain` should be call with current directory? But if so, why new domain should be defined at all? Why just not `UseLocalTranslation [boolean]` or so?

Reply:
yes we don't need to define it, we can just assume that every foo_bar.ssc may have foo_bar.mo so just do `bindtextdomain("foo_bar", current_dir)`